packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -117,6 +140,14 @@ function TabsComponent(props: Props): JSX.Element {
       blockElementScrollPositionUntilNextRender(newTab);
       setSelectedValue(newTabValue);
 
+      if (queryString) {
+        const searchKey =
+          typeof queryString === 'string' ? queryString : groupId || '';
```

Comment:
I guess we could throw if queryString is true but groupId is falsy?

Reply:
Good point, done.